webui/react/src/pages/TrialDetails/TrialInfoBox.tsx

```diff
@@ -50,6 +49,11 @@ const TrialInfoBox: React.FC<Props> = ({ trial, experiment }: Props) => {
   return (
     <Section>
       <Grid gap={ShirtSize.medium} minItemWidth={180} mode={GridMode.AutoFill}>
+        {trial.runnerState &&
+          <OverviewStats title="Last Runner State">
+            {trial.runnerState}
```

Comment:
**suggestion:** what do you think about using the `<Badge>` component instead of pure text here?

Reply:
![Screen Shot 2021-08-24 at 9 20 41 AM](https://user-images.githubusercontent.com/220971/130643975-1ffc02d5-29d8-43ca-bc06-d6450c7973a8.png)

Reply:
the text that comes through would be anything that user sets but we can stil wrap that in a badge 🤔 
let me see how a badge would look with some potential values

Reply:
I thought more about this. with our badging we usually specify meaningful colors to states. here we can't really do that since it's user defined.  wdy think?

Reply:
oh I had no idea the runner state is going to be user defined! hmmm yeah if that's the case the colors don't mean anything to them, it's purely our experiment state we expose to them. Does this mean `Runner State` will co-exist with our internal `State` or does it replace it (thinking what would happen to experiment list page and all the pages that reference states like `Active`, `Paused`, etc).

Reply:
but yeah, leaving them as how you have it (just text) makes sense to me.